quinn/src/connection.rs

```diff
@@ -1012,7 +1037,7 @@ impl ConnectionInner {
     }
 
     /// Used to wake up all blocked futures when the connection becomes closed for any reason
-    fn terminate(&mut self, reason: ConnectionError) {
+    fn terminate(&mut self, shared: &Shared, reason: ConnectionError) {
```

Comment:
I feel like inverting the argument order here would be more consistent with our style?